core/src/main/scala/scala/collection/parallel/ParSeqLike.scala

```diff
@@ -473,9 +474,9 @@ self =>
     protected[this] val pit: SeqSplitter[T]
   }
 
-  protected trait ParSeqLikeTransformer[R, Tp] extends ParSeqLikeAccessor[R, Tp] with Transformer[R, Tp]
+  protected trait ParSeqLikeTransformer[R, Tp] extends ParSeqLikeAccessor[R, Tp] with Transformer[R, Tp] { }
```

Comment:
hmm, I also corrected the self-type at line 52 in this file.

Reply:
good call. fixed